CMakeLists.txt

```diff
@@ -51,12 +103,30 @@ endif()
 
 # Find ZFP, relies on ZFP_DIR or ZFP_ROOT being set in environment.
 find_package(ZFP REQUIRED CONFIG)
+if (NOT ZFP_FOUND)
+  message (FATAL_ERROR " ZFP is Required for h5z_zfp")
+endif ()
+
+#-----------------------------------------------------------------------------
+# Dashboard and Testing Settings
+#-----------------------------------------------------------------------------
+option (BUILD_TESTING "Build h5z-zfp Unit Testing" OFF)
+if (BUILD_TESTING)
+  set (DART_TESTING_TIMEOUT 1200 CACHE STRING
```

Comment:
What is `DART` in this variable name? Related to a CTest dashboard of some kind?

Reply:
DART_TESTING_TIMEOUT is the CMake variable name that must be set to timebox tests. (yes, it refers to an old CTest dashboard)